datahub-web-react/src/app/entity/user/UserProfile.tsx

```diff
@@ -1,110 +1,411 @@
+                                                        <Tags>
+                                                            <Tag>
+                                                                {item.entity.info.displayName ? (
+                                                                    item.entity.info.displayName
+                                                                ) : (
+                                                                    <NotProvidedText />
+                                                                )}
+                                                            </Tag>
+                                                        </Tags>
+                                                    </Link>
+                                                );
+                                            })}
+                                        {groupSectionExpanded &&
+                                            groupsDetails?.relationships.length > 2 &&
+                                            groupsDetails?.relationships.map((item) => {
+                                                return (
+                                                    <Tags>
+                                                        <Tag>
+                                                            {item.entity.info.displayName ? (
+                                                                item.entity.info.displayName
+                                                            ) : (
+                                                                <NotProvidedText />
+                                                            )}
+                                                        </Tag>
+                                                    </Tags>
+                                                );
+                                            })}
+                                        {!groupSectionExpanded && groupsDetails?.relationships.length > 2 && (
+                                            <GroupsSeeMoreText
+                                                onClick={() => setGroupSectionExpanded(!groupSectionExpanded)}
+                                            >
+                                                {`+${groupsDetails?.relationships.length - 2} more`}
+                                            </GroupsSeeMoreText>
+                                        )}
+                                    </TagsSection>
+                                </GroupsSection>
+                            </UserSidebarSubSection>
+                            <EditProfileButton>
+                                <Button icon={<EditOutlined />} onClick={() => setEditProfileModal(true)}>
+                                    Edit Profile
+                                </Button>
+                            </EditProfileButton>
+                        </UserSidebar>
```

Comment:
As mentioned above, let's move UserSidebar into a separate component because there is a lot of code.

Reply:
sure

Reply:
Sure.